utils/remote.cc

```diff
@@ -2457,21 +2406,50 @@ void tr_curl_easy_cleanup(CURL* curl)
     }
 }
 
-int flush(char const* rpcurl, tr_variant* benc, RemoteConfig& config)
+[[nodiscard]] long get_timeout_secs(std::string_view req)
+{
+    if (req.find(R"("method":"blocklist-update")") != std::string_view::npos)
+    {
+        return 300L;
+    }
+
+    return 60L; /* default value */
+}
+
+[[nodiscard]] std::string serialize_payload(tr_variant const* const var, RemoteConfig const& config)
+{
+    auto serde = tr_variant_serde::json();
+    serde.compact();
+    fmt::print(stderr, "{:s}:{:d} serializing payload to {:d}\n", __FILE__, __LINE__, static_cast<int>(config.network_style));
+
+    if (config.network_style == api_compat::Style::Tr5)
```

Comment:
If your comment at https://github.com/transmission/transmission/pull/7963#discussion_r2636768174 (make `convert()` passthrough) still stands, then I think we don't need this `if` block.

Reply:
ff1364e6